### PR TITLE
Discart deleted records

### LINKED ISSUE #2: Discard deleted records
Deleted records are in fact hidden. They are marked deleted by letting their description start with the word "HIDDEN". We should properly discard such records here.

### PR DESCRIPTION
**Pull request by edgardleal**
Closes #2

### CODE REVIEW

**Comment by almarklein** [comment]
Thanks! Could you change this to use `r.get("ds", "").startswith("HIDDEN")`, and also fix the GET in `get_running_records`?

**Comment by edgardleal** [comment]
:white_check_mark: Fixed.

File: timetagger_cli/core.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def get_running_records():
     t1 = now - 35 * 60
     t2 = now + 60
     ob = request("GET", f"records?timerange={t1}-{t2}")
-    return [r for r in ob["records"] if r["t1"] == r["t2"]]
+    return [r for r in ob.get("records", []) if r.get("t1", "") == r.get("t2", "")]
```
Comment (almarklein) [inline timetagger_cli/core.py:62]: Why were these `get`'s necessary? As far as I can tell the response always included `records` and each record always has `t1` and `t2` (though `ds` is optional).

Comment (edgardleal) [reply timetagger_cli/core.py]: Yes, I agree, this is before a read the tests and understand it.

**Comment by almarklein** [comment]
Thanks!

**Pull request merged**